roles/openshift_node_upgrade/tasks/rpm_upgrade.yml

```diff
@@ -12,3 +12,8 @@
 - name: Ensure python-yaml present for config upgrade
   package: name=PyYAML state=present
   when: not openshift.common.is_atomic | bool
+
+- name: Install Node service file
+  template:
+    dest: "/etc/systemd/system/{{ openshift.common.service_type }}-node.service"
+    src: "{{ openshift.common.service_type }}-node.service.j2"
```

Comment:
We do. I have that at https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node_upgrade/tasks/restart.yml#L8-L14 which ensures that, before restarting, the daemon is reloaded. I'll go ahead and throw it in here as well as it's possible this gets included without using ``restart.yml``.